template-processors/base/Dockerfile

```diff
@@ -1,19 +1,21 @@
-FROM alpine:3.10
+FROM registry.access.redhat.com/ubi8/ubi:latest
```

Comment:
use new redhead repo

Reply:
It requires authentication, so I don't think we can do this right now.